logitech_r400/src/logitech_r400/logitech_r400

```diff
@@ -64,23 +65,30 @@ class LogitechR400Usb(object):
 
     def __exit__(self, type, value, traceback):
         for device in self._devices:
-            self.release_device(device)
-
-    def is_R400(self, dev):
-        if (dev.idVendor == R400_ID_VENDOR and dev.idProduct == R400_ID_PRODUCT) \
-           or (dev.idVendor == R400_ID_VENDOR_2 and dev.idProduct == R400_ID_PRODUCT_2):
-            return True
+            self._release_device(device)
+
+    def _read_config(self):
+        configs_file_path = rospkg.RosPack().get_path('logitech_r400') + '/config/config.yaml'
+        with open(configs_file_path, 'r') as stream:
+            self.configs = yaml.load(stream)
```

Comment:
You have declared `self._config` in the constructor, but you are not using it here. You are declaring a different member.